### DISCUSSION TITLE
Support for apple/OpenELM-3B-Instruct

### DISCUSSION THREAD

**Discussion by bks5881** [vllm-project/vllm#4350] [Ideas]
Can vllm support https://huggingface.co/apple/OpenELM-3B-Instruct if not, can that be added?

**Comment by Xuekai-Zhu** [comment]
the same question !

**Comment by stas00** [comment]
Ideally the whole family please - the `apple/OpenELM-270M-Instruct`, `apple/OpenELM-450M-Instruct` and alike in particular because they are so small - perfect candidates for dev and testing work.

Thank you!

**Comment by solume** [comment]
Really looks like its the only reasonable tiny model, would be really great to the support for this in vllm

**Comment by whr819987540** [comment]
@youkaichao  do you have any plans for this model?

**Reply by youkaichao** [reply]
contribution is welcome.
